README-dev.md

```diff
@@ -18,13 +18,13 @@ cd up42-py
 make install[dev]
 ```
 
-3. Create a new project on [UP42](https://up42.com).
+3. Follow authentication guidelines [Authenticate](/docs/authentication.md).
 
-4. Create a `config.json` file and fill in the [project credentials](https://docs.up42.com/developers/authentication#step-1-find-project-credentials).
+4. Create a `config.json` file and fill in the credentials.
 ```json
 {
-  "project_id": "...",
-  "project_api_key": "..."
+  "credentials_id": "...",
```

Comment:
We already provide config.json instructions in `authenticate.md`. This is how I previously edited the file: [link](https://github.com/up42/up42-py/pull/486/commits/09dae048100fbc5c8658b962840861eec68a10e3#diff-faeb6913d3584750b35461ea79794816d906655f7af245cdd934e118271e7cb1R21).